### DISCUSSION TITLE
change the range of the colorbar

### DISCUSSION THREAD

**Discussion by valentinamun** [OpenDrift/opendrift#1079] [General]
Hello Team, 

¿How can i  change the range of the colorbar between -200 and 0 in the graphic?
##################################
import os
import opendrift
import numpy as np

o = opendrift.open("archivo.nc",elements=np.arange(0,587160,500))
o.plot(buffer=.2, fast=True, linecolor = 'z',zmin=-200,zmax=0,filename='archivo.png')

**Comment by knutfrode** [comment]
Hi, you should use vmin and vmax, not zmin and zmax  (same as standard matplotlib)

**Reply by valentinamun** [reply]
Thanks, but I still have problems with the colorbar range using the vmin and vmax. my script are not working.

#!/usr/bin/

import os
import opendrift
import numpy as np
import os.path
from datetime import datetime, timedelta


o = opendrift.open("mosa_VM_1mes_horario_back_AS_27-31Enero.nc",elements=np.arange(0,587160,500))
o.plot(buffer=.2, vmin=-200, vmax=0, linecolor = 'z', fast=True, filename='prueba.png')

**Comment by AndresSepulveda** [comment]
Valentina, try cmin and cmax, instead of zmin zmax

**Comment by knutfrode** [comment]
Hm, it is supposed to be vmin and vmax.
Are you sure the particles really span depths from -200 to 0?

**Comment by AndresSepulveda** [comment]
Hi Knut,

It is a backwards simulation. The release points are between 0 and 200m, that is why we want to restrict the colorbar to -200m to 0m.

**Comment by knutfrode** [comment]
Ok. You can also check the actual depths with
`o.plot_property('z')`

**Comment by AndresSepulveda** [comment]
Well, some  particles do go deeper than 200m in their backward trajectory,  but our areas of interest (origin areas) are also shallow, so we want the colorbar to only spam -200 to 0m
